hawkular-alerts-ui/src/main/ui/bower.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "hawkular-alerts-ui",
+  "version": "2.0.0",
```

Comment:
Minor, this is just informational and it doesn't mean to deploy anything, but I guess we should put here a 1.8.0.

Reply:
To be honest, this manually maintained versioning is unlikely to stay updated.  But at least we can start with a reasonable version.  I'll change.